samples/portability/cmsis_rtos_v2/timer_synchronization/sample.yaml

```diff
@@ -2,7 +2,8 @@ sample:
   name: CMSIS_RTOS_V2 Synchronization
 tests:
   sample.portability.cmsis_rtos_v2.timer_synchronization:
-    tags: cmsis_rtos_v2_synchronization
+    timeout: 60
```

Comment:
If the default is 60, why would we need it expressly set here?

Reply:
Because I don't use 60 seconds as a default, it's way too long. Most tests complete in 5-10 seconds, 15 seconds default is enough.

Reply:
@marc-hb why are you doing that? sanitycheck detects when the tests complets, all changing the default to a shorter time is gonna do is increase the likelihood your run will fail

Reply:
> why are you doing that? 

Simple: when some random test that normally takes 5 seconds hangs forever, I prefer to wait 15 seconds than 60.

>  all changing the default to a shorter time is gonna do is increase the likelihood your run will fail

E pur si muove. I've been using a shorter default and the very few tests in this PR are the only ones that needed a longer timeout.

I thought it would be uncontroversial to flag the very few, long duration tests in the entire suite but forget it, this is not the point of this PR and as I said above I'll remove them (when I get back from vacation).

(Now I just remembered about the existing "slow" tag, didn't look at that yet)